client/src/components/shipping_info/style.css

```diff
@@ -0,0 +1,32 @@
+.flex-container {
+    display: flex;
+    flex-direction: column;
+    border: 1px #000000;
+  }
+  input[type="text"] {
```

Comment:
don't give the `input` style in general, give it a className and put your style there.
otherwise, you'll affect all the other `inputs` in the project

Reply:
i solve it @ishak52